src/Hl7.Fhir.Specification/Validation/Slicing/DisciminatorFactory.cs

```diff
@@ -26,6 +26,9 @@ internal static class DiscriminatorFactory
             if (spec?.Type == null) throw new ArgumentNullException(nameof(spec), "Encountered a discriminator component without a discriminator type.");
             if (resolver == null) throw Error.ArgumentNull(nameof(resolver));
 
+            //Context is needed by some external validators in case a system is missing.
```

Comment:
Nowadays, when I program such business rules, I add a link to the FHIR spec in the comment so others who wonder what the heck you are talking about can just follow the link and read up.